packages/medusa/src/api/routes/admin/reservations/create-reservation.ts

```diff
@@ -121,6 +121,7 @@ export default async (req, res) => {
  */
 export class AdminPostReservationsReq {
   @IsString()
+  @IsOptional()
```

Comment:
question: Does that mean that we support reservations that are not related to an item?

Reply:
Yes, "Custom reservations" that can exist outside of orders